src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-form"
+      action="#"
+    >
+      <div class="form__field form__field--large" data-qa="big">
+        <img
+          class="icon-search icon-search--big"
+          src="images/Search.svg"
+          alt="search icon"
+        >
+        <input
+          class="form__input form__input--big"
+          type="text"
+          name="lacation"
```

Comment:
You can write more descriptive name for input
```suggestion
          name="bigSearchInput"
```